src/Aliucord.ts

```diff
@@ -14,34 +13,31 @@ interface SettingsSchema {
     disablePluginsOnCrash: boolean;
     plugins: Record<string, boolean>;
 }
-export class Aliucord {
-    logger = new Logger("Aliucord");
-    settings!: Settings<SettingsSchema>;
 
-    debugWS = new DebugWS();
-    reactDevTools = new ReactDevTools();
+export * as pluginManager from "./api/PluginManager";
+export const logger = new Logger("Aliucord");
+export let settings: Settings<SettingsSchema>;
 
-    pluginManager: PluginManager = new PluginManager(this);
+export let load = async function loadFn() {
+    // Make sure this is only called once
+    //@ts-ignore
+    load = void 0;
```

Comment:
just set a variable, you don't need to override the function with weird js fuckery